update_api_key.py

```
@@ -1,157 +1,723 @@
+            prof, key = line.split(delimiter, 1)
+            entries.append((prof.strip(), key.strip(), source))
+        else:
+            entries.append((default_profile, line, source))
+    return entries
+
+
+def update_api_key(
+    api_key: str,
+    *,
+    profile: str = DEFAULT_PROFILE,
+    use_encryption: bool = True,
```

Ruff (0.13.2)

Unused function argument: `use_encryption`

(ARG001)